### ISSUE TITLE
Wrong release 4.0 name

### ISSUE THREAD

**Issue by socketpair** [sjvermeu/cvechecker#49]
For example,

4.0 release:
https://github.com/sjvermeu/cvechecker/archive/cvechecker-4.0.tar.gz
contains top-directory `cvechecker-cvechecker-4.0`

3.9 release
https://github.com/sjvermeu/cvechecker/archive/v3.9.tar.gz
contains top-directory `cvechecker-3.9`

**Comment by sjvermeu** [comment]
Yeah I wanted to have the archive itself named "cvechecker-4.0.tar.gz" but didn't expect that the contents will also have a different directory structure then :-( If I make a new release later on I'll revert back to the old one.

**Comment by socketpair** [comment]
Why not to make a new release (4.1) without any code changes right now ?
